controllers/users.js

```diff
@@ -0,0 +1,114 @@
+const { User } = require('../models')
+const bcrypt = require('bcryptjs')
+const jwt = require('jsonwebtoken')
+require('dotenv').config()
+
+class Controller {
+    static showAll(req,res){
+        User.findAll({order:[["id","ASC"]]})
+        .then((users)=>{
```

Comment:
Perhatikan indetasi nya masih ada yang harus diperbaiki

```suggestion
           .then((users) => {
```